lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -183,13 +183,16 @@ import Servant.API.Verbs
 
 type ApiV2 n = "v2" :> Api n
 
+-- | The full cardano-wallet API.
+--
+-- The API used in cardano-wallet-jormungandr may differ from this one.
 type Api n =
```

Comment:
I assume this one can be fully moved to each target package, alongside the server instantiation.

Reply:
No! Sadly not easily, because we have a swagger spec in core!

My plan is to keep this type, but let jormungandr use a custom `Api` type.

Reply:
Hmmm. Well no, we can also declare an "Api" type in the relevant test.

Reply:
> Well no, we can also declare an "Api" type in the relevant test

That might make our test meaningless since if it is no longer testing the `Api` type that is actually used.

I say we keep the haskell API in `core` as the most important api, and test it, but not the jormungandr one.

…or, we define and test both APIs in core 🤔 

…or, maybe one could do something with a `ApiStakePool | ApiJormungandrStakePool` sum type 🤔